Source/Core/Matrix2.js

```diff
@@ -86,6 +87,67 @@ Matrix2.unpack = function (array, startingIndex, result) {
   return result;
 };
 
+/**
+ * Flattens an array of Matrix2s into an array of components. The components
+ * are stored in column-major order.
+ *
+ * @param {Matrix2[]} array The array of matrices to pack.
+ * @param {Number[]} [result] The array onto which to store the result. If this is a typed array, it must have array.length * 4 components, else a {@link DeveloperError} will be thrown. If it is a regular array, it will be resized to have (array.length * 4) elements.
+ * @returns {Number[]} The packed array.
+ */
+Matrix2.packArray = function (array, result) {
+  //>>includeStart('debug', pragmas.debug);
+  Check.defined("array", array);
+  //>>includeEnd('debug');
+
+  const length = array.length;
+  const resultLength = length * 4;
+  if (!defined(result)) {
+    result = new Array(resultLength);
+  } else if (!Array.isArray(result) && result.length !== resultLength) {
+    throw new DeveloperError(
+      "If result is a typed array, it must have exactly array.length * 4 elements"
+    );
```

Comment:
This should be inside a debug pragma block. Looks like the `Cartesian` classes need to be updated as well.